scripts/projects.js

```diff
@@ -20,24 +20,32 @@ function Project(itm){
   this.authors = itm.authors;
 }
 
-Project.prototype.toHtml = function() {
+Project.prototype.toHtml = function(el, inx, arry) {
+  //delcare local variables
   var finAuthors = '';
+  var finProj = '';
   var authorTemplate = $('#authTemp').html();
   var projectTemplate = $('#projTemp').html();
   var dataSource = this;
-  console.log(dataSource);
   var compAuth = Handlebars.compile(authorTemplate);
   var compProj = Handlebars.compile(projectTemplate);
+  //Place the heading before the first item only
+  if(inx==0){
```

Comment:
@Ansem717 best practice is to use the explicit '===' operator when do explicit comparisons